test/games/prime/patcher_file_lib/test_item_hints.py

```diff
@@ -125,7 +125,7 @@ def test_create_hints_item_joke(empty_patches, players_config):
                                         namer, rng)
 
     # Assert
-    joke = "While walking, holding L makes you move faster."
+    joke = "ur Friend Roster is currently empty."
```

Comment:
Shouldn't this be "Your Friend Rooster" instead of "ur Friend Rooster"?

Reply:
Yeah, I goofed the copy paste haha